packages/DatePicker/src/components/ShortcutPanel/ShortcutPanel.styles.js

```diff
@@ -0,0 +1,140 @@
+import styled, { css } from "styled-components";
+import tokens from "@paprika/tokens";
+import stylers from "@paprika/stylers";
+import { HoveredItemStyles, SelectedItemStyles } from "../../shared.styles";
+
+export const ActionBarStyled = styled.div`
+  align-items: center;
+  background: ${tokens.color.blackLighten80};
+  bottom: 0;
+  border-top: 1px solid ${tokens.border.color};
+  box-sizing: border-box;
+  display: flex;
+  height: ${stylers.spacer(5)};
+  padding-left: ${tokens.spaceLg};
+  position: absolute;
+  width: 100%;
+
+  > [type="button"] {
+    flex-shrink: 0;
+
+    &:last-child {
+      margin-left: ${tokens.spaceSm};
+    }
+  }
+`;
+
+export const ColumnHeaderStyled = styled.div`
+  align-items: center;
+  background: ${tokens.color.blackLighten80};
+  border-bottom: 1px solid ${tokens.border.color};
+  display: flex;
+  flex-shrink: 0;
+  font-weight: bold;
+  height: ${stylers.spacer(5)};
+  justify-content: center;
+  padding: ${tokens.space};
+
+  ${({ isYear }) =>
+    isYear &&
+    css`
+      border-left: 1px solid ${tokens.border.color};
+      justify-content: space-between;
+    `}
+`;
+
+export const ContainerStyled = styled.div`
+  border: 1px solid ${tokens.border.color};
+  border-radius: ${tokens.border.radius};
+  box-sizing: border-box;
+  color: ${tokens.color.black};
+  font-family: ${tokens.fontFamily.default};
+  ${stylers.fontSize(-1)}
+  height: 245px;
+  overflow: hidden;
+  position: relative;
+  width: 257px;
+
+  ${({ isOpen }) =>
```

Comment:
I'd be tempted to define the two lines of css as a variable so you can do this in one clean line... but it's not _so_ bad as it is.  Your call.

Reply:
I think that makes sense to me, I will refactor it 😄